src/modality-specific-files/behavioral-experiments.md

```diff
@@ -30,7 +39,7 @@ In addition to the metadata that is either:
 -   REQUIRED for some data that can be found in the `beh` directory
     (for example `SamplingFrequency` and `StartTime` for `*_<physio|stim>.tsv.gz` files),
 
-it is RECOMMENDED to add the following metadata to the JSON files of this directory.
+It is RECOMMENDED to add the following metadata to the JSON files of this directory.
```

Comment:
The "In addition to the metadata" sentence is never completed. This is the final clause. It should not be capitalized.

Reply:
ha yes!! 

thanks